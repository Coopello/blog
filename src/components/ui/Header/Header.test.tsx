import renderer from "react-test-renderer";

import { Header } from "./";

describe("ui/Header", () => {
  it("Snap Shot", () => {
    const component = renderer.create(<Header />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
