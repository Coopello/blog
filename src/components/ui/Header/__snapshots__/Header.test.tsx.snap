// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/Header Snap Shot 1`] = `
<header
  className="flex items-center p-4"
>
  <img
    alt="こぺブログロゴ"
    data-nimg="1"
    decoding="async"
    height={60}
    loading="lazy"
    onError={[Function]}
    onLoad={[Function]}
    src="/_next/image?url=%2Fstatic%2Fcoopeblog_logo.png&w=128&q=75"
    srcSet="/_next/image?url=%2Fstatic%2Fcoopeblog_logo.png&w=64&q=75 1x, /_next/image?url=%2Fstatic%2Fcoopeblog_logo.png&w=128&q=75 2x"
    style={
      {
        "color": "transparent",
      }
    }
    width={60}
  />
  <h1
    className="text-4xl"
  >
    Coope Blog
  </h1>
</header>
`;
